templates/zerver/help/edit-or-delete-a-message.md

```diff
@@ -33,11 +33,10 @@ content.
     assuming that feature has not been
     [disabled by an organization administrator](/help/disable-message-edit-history).
```

Comment:
Let's change "You can view..." to:

> You can [view a message's edit history](https://zulip.com/help/view-a-messages-edit-history) if it is [enabled](https://zulip.com/help/disable-message-edit-history) in your organization.

It'll be easier to parse than the double negative.